Source/Core/DolphinQt2/Resources.cpp

```diff
@@ -56,10 +57,9 @@ void Resources::Init()
 	for (QString rating : ratings)
 		m_ratings.append(QPixmap(rating.prepend(sys_dir)));
 
-	QString theme_dir = QString::fromStdString(File::GetThemeDir(SConfig::GetInstance().theme_name));
-	m_misc.append(QPixmap(QStringLiteral("nobanner.png").prepend(theme_dir)));
-	m_misc.append(QPixmap(QStringLiteral("dolphin_logo.png").prepend(theme_dir)));
-	m_misc.append(QPixmap(QStringLiteral("Dolphin.png").prepend(theme_dir)));
```

Comment:
Using `theme_dir` with `dolphin_logo` and `Dolphin` seems like a mistake. The files are only in the Resources directory. Is that correct? (cc @spxtr)